public/video-ui/src/components/ManagedForm/ManagedSection.js

```diff
@@ -19,13 +19,16 @@ export class ManagedSection extends React.Component {
 
   render() {
     const hydratedChildren = React.Children.map(this.props.children, child => {
-      return React.cloneElement(child, {
-        data: this.props.data,
-        updateData: this.props.updateData,
-        updateFormErrors: this.props.updateFormErrors,
-        updateWarnings: this.props.updateWarnings,
-        editable: this.props.editable
-      });
+
+      if (child) {
```

Comment:
is this check needed? `child` should be defined within the `.map`?

Reply:
If we are creating an atom, then I'm passing in null rather than the created by field so we should just ignore that element.